### DISCUSSION TITLE
How to get units_per_em of a font

### DISCUSSION THREAD

**Discussion by floppyhammer** [yeslogic/allsorts#83] [Q&A] [answered]
I'm trying to convert GlyphPosition::hori_advance to something in pixels. I suppose I can get what I want by doing
```rs
hori_advance * font_size / units_per_em
```
But I'm struck at getting `units_per_em`. Am I missing something?

**Comment by wezm** [comment, marked as answer]
`units_per_em` is in the `head` table. If you have a [Font](https://docs.rs/allsorts/latest/allsorts/font/struct.Font.html) you can do something like this:

```rust
    let head = font.head_table()
        .expect("unable to parse head table")
        .expect("font lacks a head table");
    println!("units per em: {}", head.units_per_em);
```

**Reply by floppyhammer** [reply]
Thanks!
